src/provider/index.ts

```diff
@@ -117,7 +117,7 @@ export interface IQueuedIntent {
      * application - selector will be null until the intent reaches the front of
      * the queue.
      */
-    selector: fin.OpenFinApplication|null;
+    selector: fin.OpenFinApplication | null;
```

Comment:
The spaces around `|` is inconsistent with layouts - we worrying about that later?